### DISCUSSION TITLE
How is NavigationViewDefaultPaneBackground set in winui3 Gallery

### DISCUSSION THREAD

**Discussion by mjishnu** [microsoft/WinUI-Gallery#1453] [Q&A]
hello i am new to windows app development and was wondering how NavigationViewDefaultPaneBackground is set. i tried to search both online and read the app source but i am not able to find any clue. as of my knowledge the mica, acrylic material, backdrop and BlurAmount property of acrlicbrush  are not supported in winui3 so how is it achieving the effect. 

images:

(not asking about this:)
<img width="949" alt="Screenshot 2024-02-22 111742" src="https://github.com/microsoft/WinUI-Gallery/assets/83004520/514d31c0-826e-4bdf-a9db-4167d1ad8e4e">

(want to know how this is set:)
<img width="698" alt="Screenshot 2024-02-22 112137" src="https://github.com/microsoft/WinUI-Gallery/assets/83004520/90ada490-cd94-4b39-a59a-4b863cc1f224">

<img width="605" alt="Screenshot 2024-02-22 112147" src="https://github.com/microsoft/WinUI-Gallery/assets/83004520/e6991f08-7df9-4707-b5c5-811e4443bfea">

(sorry for the mess i was just testing out things)

**Comment by marcelwgn** [comment]
Hi, sorry for the delayed response. WinUI 3 does in fact support Acrylic and Mica backdrops, however they did not at start. With regard to NavigationView and the `NavigationViewDefaultPaneBackground`, according to the source code of WinUI 3 it is set to `AcrylicInAppFillColorDefaultBrush`. If you want to learn more about Mica and Acrylic in WinUI 3, the gallery also has dedicated pages for that 😊
